dora/core/common/src/main/java/alluxio/uri/UfsUrl.java

```diff
@@ -363,7 +357,9 @@ public int getDepth() {
    */
   public String getName() {
     List<String> pathComponents = getPathComponents();
-    Preconditions.checkArgument(!pathComponents.isEmpty());
+    if (pathComponents.isEmpty()) {
+      return "";
```

Comment:
why is path components list allowed to be empty? example?

Reply:
It is used to handle url "/". The internal class `Parser` splits raw String to pathComponents by '/', so path "/" is corresponding to an empty pathComponents. Since it is an UfsUrl, may the path like "/" be disappeared? Is it possible that UfsUrl is  local file system with path "/"?

Reply:
It is used to handle url "/". The internal class Parser splits raw String to pathComponents by '/', so path "/" is corresponding to an empty pathComponents. Since it is an UfsUrl, may the path like "/" be disappeared? Is it possible that UfsUrl represents a local file system with path "/"?